package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "covid19_scenarios",
-  "version": "1.8.1",
+  "version": "1.7.6",
   "main": "index.js",
```

Comment:
Something went wrong and you are trying to downgrade our app and all the dependencies. Looks like some merge did not go well.